examples/mediapipe/customMediaPipe/index.html

```diff
@@ -0,0 +1,227 @@
+<html>
+<head>
+	<style>
+        body{
+            margin: 0px;
+        }
+        .headline {
+            text-align: center;
+        }
+
+        vwc-top-app-bar img {
+            margin-left: 16px
+        }
+
+        .maincard{
+            margin: 10 auto;
+            width: 1134px;
+            height: fit-content;
+            display: block;
+        }
+
+        .maincardlayout{
+            display: grid;
+            grid-template-columns: 454px 674px;
+            overflow: hidden;
+        }
+
+        .sourecevideo{
+            aspect-ratio: 1;
+            width: 100%;
+            background: white;
+            border-radius: 4px;
+            height: 100%;
+            object-fit: cover;
+            border: 1px solid var(--vvd-color-neutral-30);
+        }
+        
+        .previewvideodiv{
+            aspect-ratio: 1;
+            width: 100%;
+            background-image: url(images/vonage.png);
+            border-radius: 4px;
+            height: 100%;
+            object-fit: cover;
+            border: 1px solid var(--vvd-color-neutral-30);
+        }
+        .previewvideo{
+            width: 100%;
+            border-radius: 4px;
+            height: 100%;
+            object-fit: cover;
+        }
+	</style>
+
+	<script type="module">
+            import "@vonage/vwc-top-app-bar";
+            import "@vonage/vwc-layout";
+            import "@vonage/vwc-text";
+            import "@vonage/vwc-banner";
+            import "@vonage/vwc-card";
+            import "@vonage/vwc-switch";
+            import "@vonage/vwc-select";
+            import "@vonage/vwc-list/vwc-list-item";
+	</script>
+  <title>MediaPipe sample</title>
+  <link rel="icon" href="https://vonage-client-media-samples.s3.amazonaws.com/vonage.png">
+</head>
+<body>
+<vwc-top-app-bar alternate>
+	<img slot="navigationIcon" height="24" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQQI9k8NKq3KoOs1VkwSYwPwwQbisiS_K-0jtBUyml13k7EBy8Aekz7G7I1vqceWf5kEQ&usqp=CAU"/>
+	<div>
+        <vwc-banner open connotation="info" message="Vonage MediaPipe integration sample application"></vwc-banner>
+		<vwc-layout gutters="xs" column-basis="block">
+            <vwc-text font-face="title-1" class="headline" tight>This example shows the use of this MediaPipe models:</vwc-text>
```

Comment:
Edit suggestion: _"This sample app shows how to perform media processing by using MediaPipe models along with the Vonage Media Processor and ML Transformers libraries."_

Reply:
changed it but its too long and not looks good...
do you have an idea for shorter version?

Reply:
_"Using MediaPipe models with Vonage Media Processor and ML Transformers libraries."_ ?

Reply:
_"This sample app shows how to use MediaPipe models with Vonage Media Processor and ML Transformers libraries."_ as we discussed.